crates/libs/registry/src/lib.rs

```diff
@@ -2,7 +2,7 @@
 Learn more about Rust for Windows here: <https://github.com/microsoft/windows-rs>
 */
 
-#![cfg_attr(not(test), no_std)]
+#![cfg_attr(all(not(feature = "std")), no_std)]
```

Comment:
#3102 removed the embedded tests used to generate bindings, but the new `std` feature still necessitates using the Rust Standard Library for `OsString` support.